vmaas_sync/advisory_sync.go

```diff
@@ -15,13 +15,14 @@ import (
 
 const SyncBatchSize = 1000 // Should be < 5000
 
-func syncAdvisories(modifiedSince *time.Time) error {
+func syncAdvisories(syncStart time.Time, modifiedSince *time.Time) error {
```

Comment:
Why is `syncStart` function argument and not
`syncStart := time.Now()`
as the first line in the function?

Reply:
Because I want to measure overall sync process duration. Sync advisories is only one part. Well it's the first part so it could be done like that. But I wanted to have the same interface for all sync functions. But I can change it if you want.